### PR TITLE
feat: Detect community id for wallet activity

### PR DESCRIPTION
**Pull request by Cuteivist**
Task https://github.com/status-im/status-desktop/issues/13121

Status-desktop change: https://github.com/status-im/status-desktop/pull/13627

Add community ID detection for wallet activity

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

File: services/wallet/activity/filter.sql [hunk 1]
```diff
@@ -213,7 +213,12 @@ SELECT
 	CASE
 		WHEN transfers.tx_from_address = zeroAddress AND transfers.type = "erc20" THEN (SELECT 1 FROM json_each(transfers.receipt, '$.logs' ) WHERE json_extract( value, '$.topics[0]' ) = communityMintEvent)
 		ELSE NULL
-	END AS community_mint_event
+	END AS community_mint_event,
+	CASE 
+		WHEN transfers.type = 'erc20' THEN (SELECT community_id FROM tokens WHERE transfers.token_address = tokens.address AND transfers.network_id = tokens.network_id)
+		WHEN transfers.type = 'erc721' THEN (SELECT community_id FROM collectible_data_cache WHERE transfers.token_address = collectible_data_cache.contract_address AND transfers.network_id = collectible_data_cache.chain_id)
```
Comment (IvanBelyakoff) [inline services/wallet/activity/filter.sql]: what about 1155? @alaibe do you think we should already add this case as we anticipate ERC1155 contract ready soon?

File: services/wallet/activity/activity.go [hunk 2]
```diff
@@ -492,6 +496,7 @@ func getActivityEntries(ctx context.Context, deps FilterDependencies, addresses
 		960,    // A block on layer 1 is every 12s, finalization require 64 blocks. A buffer of 16 blocks is added to not create false positives.
 		limit, offset)
 	if err != nil {
+		fmt.Println("============== QUERY ERROR", err)
```
Comment (IvanBelyakoff) [inline services/wallet/activity/activity.go]: Leftover?

**Review by IvanBelyakoff — COMMENTED** [review]
Nice job, @Cuteivist please check inline comments

**Review by stefandunca — APPROVED** [review]
It looks good to go after following on @IvanBelyakoff's comments

File: services/wallet/activity/filter.sql (hunk 1 above)
Comment (Cuteivist) [reply services/wallet/activity/filter.sql]: Fixed

File: services/wallet/activity/activity.go (hunk 2 above)
Comment (Cuteivist) [reply services/wallet/activity/activity.go]: Fixed

**Pull request merged**